src/revisions/revisions.service.ts

```diff
@@ -34,6 +34,23 @@ export class RevisionsService {
     });
   }
 
+  async purgeRevision(note: Note): Promise<Revision[]> {
```

Comment:
This function should get it's own unit test in the `revsions.service.spec.ts` file.

Reply:
Done

Reply:
I think this function should be called 
```suggestion
  async purgeRevisions(note: Note): Promise<Revision[]> {
```
or
```suggestion
  async purgeHistory(note: Note): Promise<Revision[]> {
```